autoload/codeium/command.vim

```diff
@@ -28,8 +28,18 @@ function! codeium#command#XdgConfigDir() abort
   return config_dir . '/codeium'
 endfunction
 
-function! codeium#command#HomeDir() abort
-  return $HOME . '/.codeium'
+" function! codeium#command#HomeDir() abort
+"   return $HOME . '/.codeium'
+" endfunction
+
+function! codeium#command#XdgDataDir() abort
```

Comment:
Rename this to `codeium#command#HomeDir()` and delete the commented function above.